client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -102,4 +168,6 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 	}
 }
 
-export default EditorMediaModalDetailPreviewVideoPress;
+export default connect(
+	null,
```

Comment:
Yeah, that makes sense. Removed in https://github.com/Automattic/wp-calypso/pull/11653/commits/0755e2c197304cb0823893f0ecb7bb68b5b3de66.